README.md

```diff
@@ -1,51 +1,64 @@
-# [Foundation for Sites](http://foundation.zurb.com)
+<p align="center">
+  <a href="http://foundation.zurb.com/">
+    <img src="https://user-images.githubusercontent.com/9939075/38782856-2a64a43e-40fa-11e8-89cd-e873af03b3c4.png" alt="Foundation for Sites 6" width="50%"/>
+  </a>
+</p>
+ 
+ 
+
+
+<p align="center">
+  <a href="https://foundation.zurb.com/sites/docs/installation.html"><b>Install</b></a>
+  | <a href="https://foundation.zurb.com/sites/docs">Documentation</a>
+  | <a href="https://github.com/zurb/foundation-sites/releases">Releases</a>
+  | <a href="CONTRIBUTING.md">Contributing</a>
+</p>
+
+---
+
 
 [![Build Status](https://travis-ci.org/zurb/foundation-sites.svg?branch=develop)](https://travis-ci.org/zurb/foundation-sites)
+[![dependencies Status](https://david-dm.org/zurb/foundation-sites/status.svg)](https://david-dm.org/zurb/foundation-sites)
```

Comment:
Not sure if shields.io would be better https://shields.io/

Reply:
I didn't know this website. It looks awesome. Why would it be better ?

Reply:
Because shields.io provides (cached) badges for many of these solutions through one url =)

Reply:
This should be done for all badges then. And we could talk about what badges are really useful. Will do this later in an other PR.

Reply:
Agreed.